reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -635,6 +638,27 @@ class StaticHtmlReporter : Reporter {
     }
 }
 
+private fun DIV.licenseLink(license: String) {
+    val pathUrl = getLicensePath(license) ?: return +license
+
```

Comment:
This time, too many blank lines 😉 This line can IMO go away, as we're not separating code *blocks*.